keyboard.go

```diff
@@ -144,7 +143,7 @@ func initConsole() (err error) {
 
     err = setup_term()
     if err != nil {
-        return fmt.Errorf("Error while reading terminfo data: %v", err)
+        return Errors.New("Error while reading terminfo data: " + err.Error())
```

Comment:
The errors package is lowercase (`errors.New`), and isn't being imported in any case, so this PR fails to compile.

I'm also not certain how worthwhile such a minor optimization would be.  Nearly every program in Go is going to import `fmt` at some point, either directly or via a dependency.

Reply:
I know that a lot of programs do already import fmt. But for those who dont often make a conscious decision not to do it because for them it is important to keep binary size down as much as possible. If fmt is an important part of the package and used in many places than I understand that it might be difficult or unwanted to remove fmt, but in this case it is really trivial and only used once.

Reply:
With my latest commit d31f8a0d16708f0716050cbdd8ed127cc45a3f6f I had to use `errors` package to report unrecognized escape sequences as error, so I took the opportunity to implement @JAicewizard suggestion.
There is no loss in substituting `fmt` with `errors`, while binary size is smaller, so I think it can be an optimization worth doing..